qutebrowser/browser/downloads.py

```diff
@@ -1412,3 +1061,6 @@ def get_tmpfile(self, suggested_name):
                                            suffix=suggested_name)
         self.files.append(fobj)
         return fobj
+
+
+temp_download_manager = TempDownloadManager()
```

Comment:
This belongs to `qutebrowser/app.py` and commit f6aefabf4072f00e4eb89571755f555acb0e4a10, but I can't add a review for this: You've removed the call to `temp_download_manager.cleanup()` without adding it back. In the end, the garbage collector will probably take care of it when deleting the temporary directory, but I think it's better to not rely on that and instead call `temp_download_manager.cleanup()`  on shutdown.